README.rst

```diff
@@ -2,8 +2,9 @@
 JupyterHub Idle Culler Service
 ==============================
 
-``jupyterhub-idle-culler`` provides a JupyterHub service to cull and
-shut down idle notebook servers and users on a JupyterHub deployment.
+``jupyterhub-idle-culler`` provides a JupyterHub service to identify and shut down idle or long-running Jupyter Notebook servers.
+The exact actions performed are dependent on the used spawner for the Jupyter Notebook server (e.g. the default `LocalProcessSpawner <https://jupyterhub.readthedocs.io/en/stable/api/spawner.html#localprocessspawner>`_, `kubespawner <https://github.com/jupyterhub/kubespawner>`_, or `dockerspawner <https://github.com/jupyterhub/dockerspawner>`_.
+In addition, if explicitely requested, all users whose Jupyter Notebook servers have been shut down this way are deleted as JupyterHub users from the internal database. This neither affects the authentication method which continues to allow those users to log in nor does it delete persisted user data (e.g. stored in docker volumes for dockerspawner or in persisted volumes for kubespawner).
```

Comment:
Wieeee thank you for this work!

LGTM! I just spotted the spelling of explicitly as something to update!

Reply:
The typo is fixed.